core/sail/fts/lucene-spin/src/test/java/org/eclipse/rdf4j/lucene/spin/LuceneSailSpinTest.java

```diff
@@ -291,7 +300,39 @@ public void testDistanceFunction()
 		}
 	}
 
-	public int countStatements(RepositoryConnection con)
+	/**
+	 * Try to reproduce Exception java.lang.IllegalStateException: Multiple Documents for term .... Try to add
+	 * already loaded datasets from 220-example.ttl <br/>
+	 * <code>
+	* <urn:test.org/data/rec2> a t:Data ;
+	*  rdfs:label "Sed ornare, leo" ;
+	*  t:number 2 .
+	* </code>
+	 * 
+	 * @throws Exception
+	 */
+	@Test
+	public void doubleLoading()
```

Comment:
There are no assertions in this test to ensure that it has the expected side effects.